waiter/src/waiter/util/utils.clj

```diff
@@ -133,18 +133,13 @@
 (defn stringify-elements
   [k v]
   (cond
-    (vector? v) (mapv (partial stringify-elements k) v)
-    (keyword? v) (keyword->str v)
+    (contains? #{true false nil} v) v
+    (map? v) v
+    (number? v) v
+    (sequential? v) (mapv (partial stringify-elements k) v)
     (instance? DateTime v) (du/date-to-str v)
-    (instance? UUID v) (str v)
-    (instance? Pattern v) (str v)
-    (instance? PersistentQueue v) (vec v)
-    (instance? ManyToManyChannel v) (str v)
-    (instance? Process v) (str v)
-    (instance? ValidationError v) (str v)
-    (= k :time) (str v)
-    (symbol? v) (str/join "/" ((juxt namespace name) v))
-    :else v))
+    (keyword? v) (keyword->str v)
```

Comment:
We have special handling for Joda Time `DateTime` objects (iso8601), and for keywords (preserving the namespace).